packages/fuels-abigen-macro/tests/test_projects/enum_as_input/src/main.sw

```diff
@@ -1,15 +1,15 @@
 contract;
 
 enum StandardEnum {
-    one: b256,
-    two: u32,
-    three: bool,
+    One: b256,
+    Two: u32,
+    Three: bool,
 }
 
 enum UnitEnum {
-    one: (),
-    two: (),
-    three: (),
+    One: (),
+    Two: (),
+    Three: (),
```

Comment:
Why rename these enum types?  Is it because the tests expect them to be in ClassCase?

Reply:
It's mostly because I believe that that's the conventional way to name your enum variants in Sway. I'm simply matching them against what the test expects, yes. 

The whole point of this PR is that what you write in Sway is what you get on the SDK side.